client/my-sites/patterns/components/pattern-library/style.scss

```
@@ -193,3 +193,15 @@
 		}
 	}
 }
+
+.pattern-gallery__search-all-categories {
+	border-color: currentColor;
+	color: #3858e9;
+	background: none;
+
+	&:enabled:hover,
```

```suggestion
	&:hover,
```
This fixes hover styling, which is currently messed up, since the button is rendered as an `<a>` instead of a `<button>`